app/src/main/java/ch/epfl/cs311/wanderwave/ui/theme/Color.kt

```diff
@@ -2,6 +2,8 @@ package ch.epfl.cs311.wanderwave.ui.theme
 
 import androidx.compose.ui.graphics.Color
 
+val placeholderColor = Color(0xFFFF0000)
+val signInButtonColorPlaceHolder = Color(0xFFFFA500)
```

Comment:
I don't think we should put individual component colors in the global theme, either use one of the reusable theme colors, or just define the color in the component itself